assets/src/scss/pages/post/_article-content.scss

```diff
@@ -221,20 +216,6 @@
     }
   }
 
-  .more-link {
-    @extend .btn;
-    @extend .btn-secondary;
-    @extend .btn-large;
-  }
-
-  .page-links {
```

Comment:
This one is not used at all anymore: https://github.com/search?q=org%3Agreenpeace+page-links&type=code